build/WDQS/README.md

```diff
@@ -25,15 +25,15 @@ If you can not use [RecentChanges] then you will need to reload from an RDF dump
 
 | Variable | Default | Since | Description |
 | --- | --- | --- | --- |
-| `MEMORY` | "" | 0.2.5 | Memory limit for Blazegraph |
-| `HEAP_SIZE` | "1g" | 0.2.5 | Heap size for Blazegraph |
-| `WIKIBASE_HOST` | "wikibase.svc" | 0.2.5 | Hostname of the Wikibase host |
+| `WIKIBASE_HOST` | "wikibase" | 0.2.5 | Hostname of the Wikibase host |
 | `WIKIBASE_SCHEME` | "http" | 0.2.5 | Scheme of the Wikibase host |
-| `WDQS_HOST` | "wdqs.svc" | 0.2.5 | Hostname of the WDQS host (this service) |
+| `WDQS_HOST` | "wdqs" | 0.2.5 | Hostname of the WDQS host (this service) |
```

Comment:
Previously the stated default for this, and the above WIkibase host, were not actually set to any default. No actual defaults are set and the defaults now reflect our Example compose setup which no longer sets up ".svc" aliases, but simply uses the name of the service (i.e. `wikibase`, `wdqs`, etc).

Reply:
could be resolved IMHO